packages/backend-common/src/discovery/SingleHostDiscovery.ts

```diff
@@ -27,6 +27,7 @@ import { readHttpServerOptions } from '@backstage/backend-app-api';
  * resolved to the same host, so there won't be any balancing of internal traffic.
  *
  * @public
+ * @deprecated Use `HostDiscovery` instead
  */
 export class SingleHostDiscovery implements PluginEndpointDiscovery {
```

Comment:
Let's just rename this one instead of duplicating the implementation. Then re-export with `export const SingleHostDiscovery = HostDiscovery`, and mark that as deprecated instead

Reply:
Addressed in b58a2b7